editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx

```diff
@@ -24,7 +34,14 @@ function Component(props: Props) {
 
   return (
     <Card handleSubmit={props.handleSubmit} isValid>
-      <QuestionHeader title={props.title} description={props.description} />
+      <QuestionHeader
+        title={props.title}
+        description={props.description}
+        info={props.info}
+        policyRef={props.policyRef}
+        howMeasured={props.howMeasured}
+        definitionImg={props.definitionImg}
+      />
```

Comment:
```suggestion
<QuestionHeader {...props}/>
```

I think this would work?